docker/README.md

```diff
@@ -0,0 +1,112 @@
+
+# How to use Aiakos with Docker
+
+There are several options to use Aiakos very easily using docker. These are (in order of complexity):
+
+- _"Have everything automatically done for me"_. See Section **1. The Fastest Way** (recommended).
+- _"Check the unit tests associated to the component"_. See Section **2. Run Unit Test of Aiakos**.
+- _"Check the acceptance tests are running properly"_ or _"I want to check that my Aiakos instance run properly"_ . See Section **3. Run Acceptance tests**.
+
+You do not need to do all of them, just use the first one if you want to have a fully operational Aiakos instance and maybe third one to check if your Aiakos instance run properly.
+
+You do need to have docker in your machine. See the [documentation](https://docs.docker.com/installation/) on how to do this. Additionally, you can use the proper FIWARE Lab docker functionality to deploy dockers image there. See the [documentation](https://docs.docker.com/installation/)
+
+----
+## 1. The Fastest Way
+
+Docker allows you to deploy an Aiakos container in a few minutes. This method requires that you have installed docker or can deploy container into the FIWARE Lab (see previous details about it).
+
+Consider this method if you want to try Aiakos and do not want to bother about losing data.
+
+Follow these steps:
+
+1. Download [Aiakos' source code](https://github.com/telefonicaid/fiware-aiakos) from GitHub (`git clone https://github.com/telefonicaid/fiware-aiakos.git`)
+2. `cd fiware-aiakos/docker`
+3. Using the command-line and within the directory you created type: `docker build -t fiware-aiakos -f Dockerfile .`.
```

Comment:
Yes, I know but in this way we give information to the user about from where I obtain the information. Just to let them know it.